test/rules/import-blacklist-submodules/test.ts.lint

```diff
@@ -0,0 +1,25 @@
+import { submodule } from '@blueprintjs/core';
+
+import { submodule } from '@blueprintjs/core/sub-module';
+                                            ~~~~~~~~~~~ [0]
+
+import submodule = require('@blueprintjs/core');
+
+import submodule = require('@blueprintjs/core/sub-module');
+                                             ~~~~~~~~~~~ [0]
+
+import { submodule } from '@blueprintjs/core';
+
+import { submodule } from '@blueprintjs/core/a/b/c/sub-module';
+                                            ~~~~~~~~~~~~~~~~~ [0]
+
+import sub-module = require('@blueprintjs/core');
+
+import sub-module = require('@blueprintjs/core/a/b/c/sub-module');
+                                              ~~~~~~~~~~~~~~~~~ [0]
+
+// non-static imports cannot be checked
+import {Observable} from rxjs;
```

Comment:
I introduced this test case back then. 
It's not valid typescript, but the parser consumes it. The AST can contain any expression as module specifier and tslint should not crash because it only expects string literals.